timereport/chalicelib/lib/delete.py

```diff
@@ -0,0 +1,23 @@
+import botocore.vendored.requests.api as requests
+import logging
+
+log = logging.getLogger(__name__)
+
+def delete_event(url, user_id, date):
+    """
+    URL :
+    /event/<user_id>?date=<start_date>
+    /event/<user_id>?&date=2018-10-05
+
+    :param url:
+    :param user_id
+    :param start_date
+    :return:
+    """
+    url = url + '/' + user_id
```

Comment:
Why not use f string or format to build the string?

Reply:
good point! we should update it everywhere